packages/realm/bindgen/package.json

```diff
@@ -1,8 +1,5 @@
 {
-  "name": "@realm/js-binding",
+  "name": "@realm/js-bindgen",
   "private": true,
-  "type": "module",
-  "workspaces": [
-    "packages/realm/bindgen/vendor/realm-core/"
```

Comment:
This was pointing nowhere. I know Mathias wanted to keep it around as an example on how to integrate this in another SDK, but that will probably be the least of their worries 🙂